lerna.json

```diff
@@ -6,6 +6,25 @@
     "access": "public",
     "registry": "https://registry.npmjs.org/"
   },
+  "changelog": {
+    "repo": "patternfly/patternfly-react",
+    "cacheDir": ".changelog",
+    "nextVersion": "Unreleased",
+    "labels": {
+      "breaking change :boom:": ":boom: Breaking Change",
```

Comment:
i've mapped these labels to our existing labels as well as added icons so that they appear visually consistent both in PF React and in our root `CHANGELOG.md`.